app/src/androidTest/java/com/example/sharingang/MapFragmentTest.kt

```diff
@@ -33,19 +33,41 @@ class MapFragmentTest {
     val grantPermissionRule: GrantPermissionRule =
         GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION)
 
+
+    private val itemTitle = "T"
+
     @Test
     fun itemsAddedAreDisplayedOnTheMap() {
         navigate_to(R.id.newItemFragment)
+        onView(withId(R.id.editItemTitle)).perform(
+            ViewActions.typeText(itemTitle),
+            ViewActions.closeSoftKeyboard()
+        )
         onView(withId(R.id.new_item_get_location)).perform(click())
-        Thread.sleep(3000)
+        Thread.sleep(6000)
```

Comment:
Maybe we could make a constant for the sleep time, that way it's easier to configure